docs/user-guide/skip_reconcile.md

```diff
@@ -1,7 +1,7 @@
 # Skip Application Reconcile
 
-!!! warning "Alpha Feature"
-    This is an experimental, alpha-quality feature.
+!!! warning "Alpha Feature (Since v2.7.0)"
+    This is an experimental, [alpha-quality](https://github.com/argoproj/argoproj/blob/main/community/feature-status.md#alpha).
```

Comment:
```suggestion
    This is an experimental, [alpha-quality](https://github.com/argoproj/argoproj/blob/main/community/feature-status.md#alpha) feature. It may be removed in future releases or modified in backwards-incompatible ways.
```